quiz.rb

```diff
@@ -36,7 +40,12 @@ def okay_to_drink age, drinking_age
 # below the desired temperature the method should return
 # "Fix the A/C whenever you have the chance. It's cool."
 def ac_needed current_temp, ac_working, desired_temp
-  # your code goes here
+  ac_working == true, current_temp > desired_temp
+  return "Turn on the AC please"
+  ac_working == false, current_temp > desired_temp
```

Comment:
Thinking here is correct too, but you need to use `&&` instead of commas. `&&` will make the block of code underneath fire only if both conditions are true